themes/bootstrap3/templates/admin/feedback/home.phtml

```diff
@@ -0,0 +1,153 @@
+          <?php foreach($this->uniqueSites as $site):?>
+            <option value="<?=$this->escapeHtmlAttr($site)?>"<?php if ($site === ($this->params['site_url'] ?? null)): ?> selected="selected"<?php endif;?>>
+              <?=$this->escapeHtml($site) ?>
+            </option>
+          <?php endforeach;?>
+        </select>
+      </label>
+      <label for="status">
+        <?=$this->translate('Status')?>
+        <select name="status" id="status" class="form-control">
+          <option value="ALL"><?=$this->translate('All')?></option>
+          <?php foreach ($this->statuses as $status): ?>
+            <option value="<?=$this->escapeHtmlAttr($status)?>"<?php if ($status == ($this->params['status'] ?? null)): ?> selected="selected"<?php endif;?>>
+              <?=$this->transEsc('feedback_status_' . $status)?>
+            </option>
+          <?php endforeach; ?>
+        </select>
+
+      </label>
+      <label for="feedbacksubmit">
+        <input type="submit" id="feedbacksubmit" value="<?=$this->transEscAttr('Filter')?>" class="btn btn-primary" />
+      </label>
+      <?php if ((null !== ($this->params['form_name'] ?? null)) || (null !== ($this->params['site_url'] ?? null)) || (null !== ($this->params['status'] ?? null))):?>
+        <a class = "btn btn-link" href="<?=$this->url('admin/feedback'); ?>"><?=$this->translate('clear_feedback_filter')?></a>
+      <?php endif;?>
+    </div>
+  </form>
+
+  <?php if (count($this->feedback) > 0):?>
+    <form action="<?=$this->url('admin/feedback', ['action' => 'Delete'])?>" method="post">
+      <input type="hidden" name="form_name" value="<?=$this->escapeHtmlAttr($this->params['form_name'] ?? '') ?>" />
+      <input type="hidden" name="site_url" value="<?=$this->escapeHtmlAttr($this->params['site_url'] ?? '') ?>" />
+      <input type="hidden" name="status" value="<?=$this->escapeHtmlAttr($this->params['status'] ?? '') ?>" />
+      <table class="table table-striped">
+        <tr>
+          <th><?=$this->transEsc('Form name')?></th>
+          <th><?=$this->transEsc('Site URL')?></th>
+          <th><?=$this->transEsc('Message')?></th>
+          <th><?=$this->transEsc('Additional data')?></th>
+          <th><?=$this->transEsc('Status')?></th>
+        </tr>
+        <?php foreach ($this->feedback as $feedbackItem): ?>
+          <?php
+            $data = json_decode($feedbackItem->form_data, true);
+          ?>
+          <tr>
+            <td>
+              <input id="checkbox_<?=$this->escapeHtmlAttr($feedbackItem->id)?>" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($feedbackItem->id)?>" class="checkbox_ui"/>
+              <input type="hidden" name="idsAll[]" value="<?=$this->escapeHtmlAttr($feedbackItem->id)?>" />
+              <?=$this->escapeHtml($feedbackItem->form_name)?>
+            </td>
+            <td><?=$this->escapeHtml($feedbackItem->site_url)?></td>
+            <td><?=$this->escapeHtml($feedbackItem->message)?></td>
```

Comment:
This value can potentially be very long. Would it make sense to limit it in some way to keep the size of the overall table under control?

Reply:
I've add truncation to 100 characters and more/less link after the text.